### PR TITLE
Add postbuster api collection

### LINKED ISSUE #121: Add api collection for Postbuster
Add the collection of API shared by ICICI so that those can be imported as PostBuster collection

### PR DESCRIPTION
**Pull request by vivesing**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #121

Test URLs:
- Before: https://main--icicidirect--aemsites.hlx.live/research/equity
- After: https://apicollection--icicidirect--aemsites.hlx.live/research/equity

- [ ] New Blocks introduced in this PR
      If yes, please provide details below

Block Name    | Documentation
------------- | -------------
 For e.g. carousel | N/A
 
 The current set of APIs 
 
image

### CODE REVIEW

**Comment by anagarwa** [comment]
@vivesing  can you please confirm if this can json can be loaded in postman as well ?

**Review by ravverma — APPROVED** [review]

File: tools/api-collection/cloudflare-collection.json [hunk 1]
```diff
@@ -0,0 +1,208 @@
+{
+  "_type": "export",
+  "__export_format": 4,
+  "__export_date": "2024-04-09T10:20:24.174Z",
+  "__export_source": "insomnia.desktop.app:v2023.5.7-adobe",
```
Comment (anagarwa) [inline tools/api-collection/cloudflare-collection.json:5]: Nit: Are these necessary field ?

Comment (vivesing) [reply tools/api-collection/cloudflare-collection.json:5]: I just exported the collection from the postbuster with default settings, it should be okay as its working fine on both postbuster and postman

**Comment by JiangLong2019** [comment]
Looks like this url can be accessed from end user, do we expect this?

https://apicollection--icicidirect--aemsites.hlx.live/tools/api-collection/cloudflare-collection.json

**Comment by sdmcraft** [comment]
I think we can move this over to Sharepoint internal folder to avoid leaking api details unless needed.

**Comment by sdmcraft** [comment]
@vivesing , should we close this PR?

**Pull request closed without merging**